Create _community-stappenplan.mdx (#964)
nl-design-system/documentatie: docs/componenten/_community-stappenplan.mdx

Doel: Het component mag door andere organisaties gebruikt worden. Producten die ermee gemaakt worden hoeven niet per se open source te zijn.

Binnen het NL Design System werken we voor componenten met de [Openbare Licentie van de Europese Unie (EUPL-1.2)](https://nldesignsystem.nl/open-source/eupl/).

Zorg ervoor dat jouw organisatie naar de EUPL-1.2 licentie verwijst op de onderstaande posities. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

- **GitHub repository**: Als één van de repository details onder 'About' in de sidebar.
- **Package in npm**: Onder 'License' in de sidebar.
- **Storybook**: Als een aparte pagina binnen de sidebar, of binnen de introductiepagina danwel README.
- **Figma** (optioneel): Op de cover of direct naast de cover van de Figma bibliotheek.

**🚩 Checkpoint**: EUPL-1.2 licentie toegepast.

## Licentie documentatie

Doel: De documentatie van een component mag door andere organisaties hergebruikt of gecombineerd worden zonder enige bronvermelding.

Binnen het NL Design System werken we voor de documentatie met de [Creative Commons 0 licentie (CC0)](https://nldesignsystem.nl/open-source/cc0).

> Zorg ervoor dat jouw organisatie de CC0 licentie benoemt in het README.md bestand van het component in Github. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer: ```suggestion
Zorg ervoor dat jouw organisatie de CC0 licentie benoemt in het README.md bestand van het component in GitHub. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

**🚩 Checkpoint**: Beschikbaar in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/).

## Visuele regressietests

Doel: Het component wordt voorzien van het Voorbeeld thema, en organisaties worden geïnformeerd wanneer aanpassingen in de component gevolgen hebben voor hun thema.

Deze stap kan alleen worden uitgevoerd door het kernteam.

- Component wordt voorzien van het Voorbeeld thema.
- Communiceren dat men het component kan gaan gebruiken en dat daar automatisch visuele regressietests op zullen plaatsvinden.

**🚩 Checkpoint**: Visuele regressietests zijn beschikbaar in de Thema Storybook.

## Figma bibliotheek

Doel: Het component wordt beschikbaar in de [NL Design System Figma bibliotheek](https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=450-4121&t=AKzhgDPYAKNI1gaT-1). Designers van andere organisaties kunnen het component overnemen en gaan gebruiken.

[Neem contact op met het kernteam](/project/kernteam) om deze stap uit te voeren.

> Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Is er geen Figma component dan ontwikkelt het kernteam deze zelf.

Reviewer: ```suggestion
Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Als er geen Figma component is, dan ontwikkelt het kernteam dit zelf.
```

**🚩 Checkpoint**: Beschikbaar in de NL Design System Component assessment (Figma).

## Status naar Community

Doel: iedereen kan zien dat het component nu richting Candidate wil.

Deze stap kan enkel worden uitgevoerd door het kernteam.

- Verander het 'Help Wanted' label van het backlog issue naar 'Community'.
- Verander het 'Help Wanted' label van de GitHub Discussion naar 'Community'.
- Filter het [Candidate bord](https://github.com/orgs/nl-design-system/projects/29/views/1) op het component door op `Component: {naam-component}` te zoeken.
- Kopieer de URL na het filteren.
- Voeg onderstaande tekst toe als comment aan de GitHub Discussion.

```md
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[{naam-component} op het Candidate bord]({url-candidate-bord})
```

**🚩 Checkpoint**: Status bijgewerkt naar Community.

## Promotie

Doel: Credits geven aan mensen die hebben bijgedragen vanuit de community. De rest van community informeren en enthousiasmeren om ook naar Community, Candidate en Hall of Fame toe te werken.

Deze stap kan enkel worden uitgevoerd door het kernteam.

### Afbeelding maken

> Het kernteam maakt een afbeelding waarbij het component in 2 thema's wordt getoond. 1 thema op basis van een huisstijl uit de community en 1 keer met het Voorbeeld thema.

Reviewer: ```suggestion
Het kernteam maakt een afbeelding waarbij het component in twee thema's wordt getoond: één thema gebaseerd op een huisstijl uit de community en één keer met het Voorbeeld thema.
```

# Stappenplan: Community

> Volg dit stappenplan om een component van alle checkspoints 🚩 te voorzien die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done).

Reviewer 2: ```suggestion
Volg dit stappenplan om een component van alle checkpoints 🚩 te voorzien die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done).
```

Heb je een vraag? Twijfel niet en stel je vraag via het [#nl-design-system Slack kanaal](https://praatmee.codefor.nl/) van CodeForNL of [neem contact op met het kernteam](/project/kernteam).

## CSS

Doel: Het component kan ingezet worden in een framework naar keuze.

Het component moet beschikbaar zijn in code. Minimaal als HTML/CSS.

> Ontwikkel het component zelf of controleer de Storybook en/of Github omgevingen van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer 2: ```suggestion
Ontwikkel het component zelf of controleer de Storybook en/of GitHub omgevingen van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

**🚩 Checkpoint**: Beschikbaar in CSS.

## Publiek beschikbaar

Doel: Men kan het component vinden, bekijken, gebruiken, forken en verbeteren.

Het component moet publiek beschikbaar zijn in Storybook.

Plaats het component in de Storybook omgeving van jouw organisatie. Of controleer de Storybook omgevingen van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

De Storybook omgeving moet te bekijken zijn zonder te hoeven inloggen.

> **🚩 Checkpoint**: Publiek in beschikbaar in Storybook.

Reviewer 2: ```suggestion
**🚩 Checkpoint**: Publiek beschikbaar in Storybook.
```

> **🚩 Checkpoint**: Publiek in beschikbaar in Storybook.

Reviewer 3: ```suggestion
**🚩 Checkpoint**: Publiek beschikbaar in Storybook.
```

## Implementatie

Doel: Het component is het component dat we als kernteam verwachten.

In de GitHub Discussion van het component staan de naam en het doel genoteerd. Check of de implementatie van jouw  organisatie hieraan voldoet. Of controleer de implementaties van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

> Noteer de implementatie bijvoorbeeld `utrecht-paragraph` of `ams-paragraph` in de kolom voor 'Implementaite'.

Reviewer 3: ```suggestion
Noteer de implementatie bijvoorbeeld `utrecht-paragraph` of `ams-paragraph` in de kolom voor 'Implementatie'.
```

> Noteer de implementatie bijvoorbeeld `utrecht-paragraph` of `ams-paragraph` in de kolom voor 'Implementaite'.

Reviewer 2: ```suggestion
Noteer de implementatie, bijvoorbeeld `utrecht-paragraph` of `ams-paragraph`, in de kolom voor 'Implementaite'.
```

> **Tip!** Met implementatie bedoelen we bijvoorbeeld `utrecht-button` of `ams-button`. Een specifiek type van een implementatie, bijvoorbeeld `zonnedael-sparkly-button` noemen we een variant.

Reviewer 2: ```suggestion
**Tip!** Met implementatie bedoelen we bijvoorbeeld `utrecht-button` of `ams-button`. Een specifiek type van een implementatie, bijvoorbeeld `zonnedael-sparkly-button`, noemen we een variant.
```

**🚩 Checkpoint**: Implementatie voldoet aan naam en doel uit de GitHub Discussion.

## Minimale design tokens

Doel: Meerdere organisaties kunnen het component naar hun huisstijl omzetten.

Door met [design tokens](/handboek/design-tokens/) te werken zorgen we ervoor dat meerdere organisaties het component van een eigen huisstijl kunnen voorzien. Minimaal zouden er design tokens beschikbaar moeten zijn om kleur en typografie beslissingen door te voeren.

> Zorg ervoor dat het component van jouw organisatie gebruik maakt van design tokens. Of controleer de implentaties van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer 3: ```suggestion
Zorg ervoor dat het component van jouw organisatie gebruik maakt van design tokens. Of controleer de implementaties van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

> **Tip!** Om te controleren of er design tokens zijn toegepast kun je gebruik maken van de 'Inspect' functionaliteit van je browser. Wanneer je in de CSS verwijzingen ziet naar CSS variablen zoals bijvoorbeeld `--denhaag-button-primary-action-background-color` of `--ams-button-font-family` kun je er vanuit gaan dat er design tokens zijn geïmplementeerd.

Reviewer 3: ```suggestion
**Tip!** Om te controleren of er design tokens zijn toegepast kun je gebruik maken van de 'Inspect' functionaliteit van je browser. Wanneer je in de CSS verwijzingen ziet naar CSS variabelen zoals bijvoorbeeld `--denhaag-button-primary-action-background-color` of `--ams-button-font-family` kun je er vanuit gaan dat er design tokens zijn gebruikt.
```

> **🚩 Checkpoint**: Implementatie heeft minimaal kleur en typografie beslissingen met design tokens geïmplementeerd.

Reviewer 3: ```suggestion
**🚩 Checkpoint**: Implementatie heeft minimaal kleur en typografie beslissingen met design tokens gebruikt.
```

> **🚩 Checkpoint**: Implementatie heeft minimaal kleur en typografie beslissingen met design tokens geïmplementeerd.

Reviewer 2: Implementatie en geïmplementeerd voelt dubbel aan in deze zin. Kunnen we bijvoorbeeld zeggen:
"Implementatie heeft minimaal kleur en typografie beslissingen met design tokens toegepast."?
Het woord toegepast heb je hierboven ook gebruikt.

Reviewer: ```suggestion
**🚩 Checkpoint**: Implementatie heeft minimaal kleur en typografie beslissingen met design tokens toegepast.
```

## Naamgeving design tokens

Doel: Een voorspelbare naamgeving van design tokens.

Voor NL Design System zijn er afspraken gemaakt voor duidelijke en voorspelbare namen. De beschikbare design tokens van een component moeten voldoen aan [de naamgeving conventie van NL Design System](https://www.nldesignsystem.nl/handboek/design-tokens/#naamgeving). Zo is een component herbruikbaar voor alle verschillende thema's.

> Zorg ervoor dat de design tokens van het component van jouw organisatie voldoen aan de naamgeving concentie. Of controleer de implentaties van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer 3: ```suggestion
Zorg ervoor dat de design tokens van het component van jouw organisatie voldoen aan de naamgeving conventie. Of controleer de implementaties van organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

**🚩 Checkpoint**: Gebruikte design tokens voldoen aan naamgeving conventie van NL Design System.

## Licentie component

Doel: Het component mag door andere organisaties gebruikt worden. Producten die ermee gemaakt worden hoeven niet per se open source te zijn.

Binnen het NL Design System werken we voor componenten met de [Openbare Licentie van de Europese Unie (EUPL-1.2)](https://nldesignsystem.nl/open-source/eupl/).

Zorg ervoor dat jouw organisatie naar de EUPL-1.2 licentie verwijst op de onderstaande posities. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

- **Github repository**: Als één van de repository details onder 'About' in de sidebar.
- **Package in npm**: Onder 'License' in de sidebar.

> - **Storybook**: Als een aparte pagina binnen de sidebar of binnen de introductiepagina danwel README.

Reviewer 2: ```suggestion
- **Storybook**: Als een aparte pagina binnen de sidebar, of binnen de introductiepagina danwel README.
```

- **Figma** (optioneel): Op de cover of direct naast de cover van de Figma bibliotheek.

**🚩 Checkpoint**: EUPL-1.2 licentie toegepast.

## Licentie documentatie

Doel: De documentatie van een component mag door andere organisaties hergebruikt of gecombineerd worden zonder enige bronvermelding.

> Binnen het NL Design System werken we voor documentatie met de [Creative Commons 0 licentie (CC0)](https://nldesignsystem.nl/open-source/cc0).

Reviewer 2: ```suggestion
Binnen het NL Design System werken we voor de documentatie met de [Creative Commons 0 licentie (CC0)](https://nldesignsystem.nl/open-source/cc0).
```

> Zorg ervoor dat jouw organisatie de CC0 licentie benoemd in het README.md bestand van het component in Github. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer 3: ```suggestion
Zorg ervoor dat jouw organisatie de CC0 licentie benoemt in het README.md bestand van het component in Github. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

> Zorg ervoor dat jouw organisatie de CC0 licentie benoemd in het README.md bestand van het component in Github. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

Reviewer 2: ```suggestion
Zorg ervoor dat jouw organisatie de CC0 licentie benoemt in het README.md bestand van het component in GitHub. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.
```

**Tip!** Door de 'Code' of 'Raw' weergave van het README.md bestand te bekijken zou je bovenaan `<!-- @license CC0-1.0 -->` moeten zien staan.

**🚩 Checkpoint**: Documentatie heeft de Creative Commons 0 licentie (CC0).

## Prefix

Doel: Organisaties kunnen componenten naast elkaar gebruiken zonder 'clashes'. Developers en designers kunnen zien waar het component vandaan komt.

Om componenten van elkaar te kunnen gebruiken is het toevoegen van een prefix zoals bijvoorbeeld `utrecht-` of `ams-` verplicht.

Zorg ervoor dat de prefix van jouw organisatie is toegevoegd aan onderstaande onderdelen. Of controleer de organisaties die aan de GitHub Discussion van het component zijn toegevoegd.

- CSS class selectors.
- Design tokens in JSON.
- CSS variables.

**🚩 Checkpoint**: API's hebben een prefix van de verantwoordelijke organisatie.

## Storybook met NL Design System thema's

Doel: Het component wordt beschikbaar voor andere organisaties.

[Neem contact op met het kernteam](/project/kernteam) om deze stap uit te voeren.

> Het kernteam maakt het component beschikbaar in de ['Themes' repository](https://github.com/nl-design-system/themes) waardoor deze is terug te vinden in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/). Andere organisaties kunnen het component gaan gebruiken.

Reviewer 2: ```suggestion
Het kernteam maakt het component beschikbaar in de ['Themes' repository](https://github.com/nl-design-system/themes), waardoor deze terug te vinden is in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/). Andere organisaties kunnen vervolgens het component gaan gebruiken.
```

**🚩 Checkpoint**: Beschikbaar in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/).

## Visuele regressietests

> Doel: Het component wordt voorzien van het Voorbeeld thema en organisaties worden geïnformeerd.

Reviewer 3: ```suggestion
Doel: Het component wordt voorzien van het Voorbeeld thema, en organisaties worden geïnformeerd wanneer aanpassingen in de component gevolgen hebben voor hun thema.
```

Deze stap kan alleen worden uitgevoerd door het kernteam.

- Component wordt voorzien van het Voorbeeld thema.

> - Communiceren dat men het component kan gaan gebruiken en dat daar automatisch visuele regressietests op zullen plaats vinden.

Reviewer 2: ```suggestion
- Communiceren dat men het component kan gaan gebruiken en dat daar automatisch visuele regressietests op zullen plaatsvinden.
```

**🚩 Checkpoint**: Visuele regressietests zijn beschikbaar in de Thema Storybook.

## Figma bibliotheek

Doel: Het component wordt beschikbaar in de [NL Design System Figma bibliotheek](https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=450-4121&t=AKzhgDPYAKNI1gaT-1). Designers van andere organisaties kunnen het component overnemen en gaan gebruiken.

[Neem contact op met het kernteam](/project/kernteam) om deze stap uit te voeren.

> Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Is er geen Figma component dan ontwikkeld het kernteam deze zelf.

Reviewer 3: ```suggestion
Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Is er geen Figma component dan ontwikkelt het kernteam deze zelf.
```

> Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Is er geen Figma component dan ontwikkeld het kernteam deze zelf.

Reviewer 2: ```suggestion
Het kernteam neemt het component over uit de Figma omgeving van jouw organisatie, inclusief bijbehorende design tokens, en plaatst deze in de NL Design System Figma bibliotheek. Daarbij wordt het component voorzien van het Voorbeeld thema. Als er geen Figma component is, dan ontwikkelt het kernteam dit zelf.
```

> Organisaties die de NL Design System Figma bibliotheek vanaf dat moment dupliceren krijgen het component direct meegeleverd. Organisaties die de NL Design System Figma bibliotheek al hadden gedupliceerd worden tijdens de volgende 'Bieb sync video' meegenomen hoe ze dit component ook kunnen overnemen.

Reviewer 2: ```suggestion
Organisaties die de NL Design System Figma bibliotheek vanaf dat moment dupliceren, krijgen het component direct meegeleverd. Organisaties die de NL Design System Figma bibliotheek al hadden gedupliceerd, worden tijdens de volgende 'Bieb sync video' geïnformeerd over hoe ze dit component ook kunnen overnemen.
```

**🚩 Checkpoint**: Beschikbaar in de NL Design System Figma bibliotheek.

## Component index

> Doel: Op nldesignsystem.nl zijn er verwijzingen naar het component met Voorbeeld thema beschikbaar.

Reviewer 3: ```suggestion
Doel: Op nldesignsystem.nl zijn er verwijzingen beschikbaar naar het component met het Voorbeeld thema.
```

Deze stap kan alleen worden uitgevoerd door het kernteam.

> Het kernteam vult de Component index aan met links naar het component met Voorbeeld thema. We verwijzen hierbij naar het component in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/) en de [NL Design System Figma bibliotheek](https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=450-4121&t=AKzhgDPYAKNI1gaT-1).

Reviewer 3: ```suggestion
Het kernteam vult de Component index aan met links naar het component met het Voorbeeld thema. We verwijzen hierbij naar het component in de [Storybook met alle NL Design System thema's](https://nl-design-system.github.io/themes/) en de [NL Design System Figma bibliotheek](https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=450-4121&t=AKzhgDPYAKNI1gaT-1).
```

**🚩 Checkpoint**: Storybook en Figma links zijn toegevoegd aan de component index.

## Component assessment

> Doel: Organisaties worden bij een onboarding snel naar de juiste component verwezen.

Reviewer 2: ```suggestion
Doel: Organisaties worden bij een onboarding snel naar het juiste component verwezen.
```

Deze stap kan alleen worden uitgevoerd door het kernteam.

> Het kernteam voegt een 'Component sticker' toe aan de Figma file '[NL Design System Component assessment](https://www.figma.com/design/NLbNbckFo9manEihRFeJmR/NL-Design-System---Component-Assessment?m=auto&t=J0gWvRWoyCJhSqot-6)'. Deze sticker linkt naar de documentatiepagina van het component op nldesignsystem.nl.

Reviewer 2: ```suggestion
Het kernteam voegt een 'Component sticker' toe aan het Figma bestand '[NL Design System Component assessment](https://www.figma.com/design/NLbNbckFo9manEihRFeJmR/NL-Design-System---Component-Assessment?m=auto&t=J0gWvRWoyCJhSqot-6)'. Deze sticker linkt naar de documentatiepagina van het component op nldesignsystem.nl.
```

> Door deze 'Component stickers' op screeshots te plakken kan het kernteam geinteresseerde organisaties inzicht bieden welke componenten er beschikbaar zijn om hun website of applicatie mee op te bouwen.

Reviewer 3: ```suggestion
Door deze 'Component stickers' op screeshots te plakken kan het kernteam geïnteresseerde organisaties inzicht bieden welke componenten beschikbaar zijn om hun website of applicatie mee op te bouwen.
```

> Door deze 'Component stickers' op screeshots te plakken kan het kernteam geinteresseerde organisaties inzicht bieden welke componenten er beschikbaar zijn om hun website of applicatie mee op te bouwen.

Reviewer 2: ```suggestion
Door deze 'Component stickers' op screenshots te plakkenm kan het kernteam geïnteresseerden organisaties inzicht bieden welke componenten beschikbaar zijn om hun website of applicatie mee op te bouwen.
```

**🚩 Checkpoint**: Beschikbaar in de NL Design System Component assessment (Figma).

## Status naar Community

Doel: iedereen kan zien dat het component nu richting Candidate wil.

Deze stap kan enkel worden uitgevoerd door het kernteam.

- Verander het 'Help Wanted' label van het backlog issue naar 'Community'.
- Verander het 'Help Wanted' label van de GitHub Discussion naar 'Community'. 
- Filter het [Candidate bord](https://github.com/orgs/nl-design-system/projects/29/views/1) op het component door op `Component: {naam-component}` te zoeken.

> - Kopieer de url na het filteren.

Reviewer 3: ```suggestion
- Kopieer de URL na het filteren.
```

- Voeg onderstaande tekst toe als comment aan de GitHub Discussion.

```md
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[{naam-component} op het Candidate bord]({url-candidate-bord})
```

**🚩 Checkpoint**: Status bijgewerkt naar Community.

## Promotie

Doel: Credits geven aan mensen die hebben bijgedragen vanuit de community. De rest van community informeren en enthousiasmeren om ook naar Community, Candidate en Hall of Fame toe te werken.

Deze stap kan enkel worden uitgevoerd door het kernteam.

### Afbeelding maken

> Het kernteam maakt een afbeelding waarbij het component in 2 thema's wordt getoond. 1 thema op basis van een huisstijl uit de communit en 1 keer met het Voorbeeld thema.

Reviewer 3: ```suggestion
Het kernteam maakt een afbeelding waarbij het component in 2 thema's wordt getoond. 1 thema op basis van een huisstijl uit de community en 1 keer met het Voorbeeld thema.
```

> Het kernteam maakt een afbeelding waarbij het component in 2 thema's wordt getoond. 1 thema op basis van een huisstijl uit de communit en 1 keer met het Voorbeeld thema.

Reviewer 2: ```suggestion
Het kernteam maakt een afbeelding waarbij het component in twee thema's wordt getoond: één thema gebaseerd op een huisstijl uit de community en één keer met het Voorbeeld thema.
```

### Slack

> - Zeg het volgende in #nl-design-system

Reviewer 2: ```suggestion
- Zeg het volgende in #nl-design-system.
```

```md
 ✨ {naam-component} is nu Community ✨

[Bekijk {naam-component} op nldesignsystem.nl]({link-naar-component-op-nldesignsystem.nl})

> Help ons om het component naar Candidate te krijgen door ons in de [GitHub Discussion voor het {naam-component} component]({link-naar-discussion}) te laten weten waar je hem gebruikt.

Reviewer 2: 'Het component' is een onzijdig woord, daarom zou ik liever 'het' gebruiken in plaats van 'hem'.

```

### Communityleden selecteren voor credits

> Het kernteam selecteert communityleden om credits aan uit te delen. Denk bijvoorbeeld aan de maintainer van Storybook en betrokken designer.

Reviewer 2: ```suggestion
Het kernteam selecteert communityleden om credits aan uit te delen. Denk bijvoorbeeld aan de maintainer van Storybook en de betrokken designer.
```

### Heartbeat, Nieuwsbrief en LinkedIn

> Het kernteam benaderd de organisatie die het component heeft ontwikkeld. We vragen of zij het component tijdens een Heartbeat willen preseteren. Mocht dit niet mogelijk zijn doen wij dit als kernteam zelf.

Reviewer 3: ```suggestion
Het kernteam benadert de organisatie die het component heeft ontwikkeld. We vragen of zij het component tijdens een Heartbeat willen preseteren. Mocht dit niet mogelijk zijn doen wij dit als kernteam zelf.
```

> Het kernteam benaderd de organisatie die het component heeft ontwikkeld. We vragen of zij het component tijdens een Heartbeat willen preseteren. Mocht dit niet mogelijk zijn doen wij dit als kernteam zelf.

Reviewer 2: ```suggestion
Het kernteam benadert de organisatie die het component heeft ontwikkeld. We vragen of zij het component tijdens een Heartbeat willen preseteren. Mocht dit niet mogelijk zijn, dan doet het kernteam dit zelf.
```

> Daarnaast plaatsen we een bericht in de nieuwsbrief en op LinkedIn. Waarin we de volgende elementen in openemen:

Reviewer 2: ```suggestion
Daarnaast plaatsen we een bericht in de nieuwsbrief en op LinkedIn. Daarin nemen we de volgende elementen in op:
```

> - Link naar component op nldesignsystem.nl

Reviewer 2: ```suggestion
- Link naar component op nldesignsystem.nl.
```

- Link naar GitHub Discussion.
- Credits
- Eventueel in combinatie met een kort interview.

**🚩 Checkpoint**: Gebruik van component uit de community gepromoot.

## 🏁 Finish

> Zo wat een werk! Je hebt alle stappen genomen en zo alle checkspoints kunnen afvinken die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). Het component gaat nu door voor de 'Candidate' status.

Reviewer 3: ```suggestion
Zo wat een werk! Je hebt alle stappen genomen en je hebt alle checkpoints kunnen afvinken die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). Het component gaat nu door voor de 'Candidate' status.
```

> Zo wat een werk! Je hebt alle stappen genomen en zo alle checkspoints kunnen afvinken die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). Het component gaat nu door voor de 'Candidate' status.

Reviewer 2: ```suggestion
Zo wat een werk! Je hebt alle stappen genomen en alle checkpoints afgevinkt die nodig zijn voor de ['Community' status](https://github.com/orgs/nl-design-system/projects/29/views/1) van het [Estafettemodel](https://www.nldesignsystem.nl/componenten/definition-of-done). Het component gaat nu door voor de 'Candidate' status.
```
